test/crypto/keys.js

```diff
@@ -16,20 +16,20 @@ import {
 	getPrivateAndPublicKeyFromSecret,
 	getRawPrivateAndPublicKeyFromSecret,
 	getKeys,
-	getAddressFromSecret,
+	getAddressAndPublicKeyFromSecret,
 } from '../../src/crypto/keys';
 import {
 	bufferToHex,
 } from '../../src/crypto/convert';
 
 describe('keys', () => {
 	const defaultSecret = 'secret';
+	const expectedPublicKey = '5d036a858ce89f844491762eb89e2bfbd50a4a0a0da658e4b2628b25b117ae09';
+	const expectedPrivateKey = '2bb80d537b1da3e38bd30361aa855686bde0eacd7162fef6a25fe97bf527a25b5d036a858ce89f844491762eb89e2bfbd50a4a0a0da658e4b2628b25b117ae09';
 	const defaultAddress = {
```

Comment:
Now that we're calling the function `getAddressAndPublicKeyFromSecret`, we should name this accordingly. Also let's make our use of `expectedVar` vs `defaultVar` consistent.